spec/support/page_objects/edit_user.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module PageObjects
+  class EditUser
+    include Capybara::DSL
+    include Rails.application.routes.url_helpers
+
+    def visit_page
+      visit edit_user_registration_path
+
+      self
```

Comment:
I don't think we need to return anything. it's just an action to do something (visit the page). I don't like chaining things here